primitive-types/Cargo.toml

```diff
@@ -11,10 +11,10 @@ rust-version = "1.60.0"
 
 [dependencies]
 fixed-hash = { version = "0.8", path = "../fixed-hash", default-features = false }
-uint = { version = "0.10.0", path = "../uint", default-features = false }
-impl-serde = { version = "0.5.0", path = "impls/serde", default-features = false, optional = true }
-impl-codec = { version = "0.6.0", path = "impls/codec", default-features = false, optional = true }
```

Comment:
`impl-codec@0.6.0` was wrong since the macro generates code that uses the latest breaking changes